model/export/AllBookletsExport.php

```diff
@@ -603,27 +606,24 @@ protected function getRows()
             foreach($results as $result){
                 /** @var DeliveryExecution $execution */
                 $execution = $this->getServiceLocator()->get(ServiceProxy::SERVICE_ID)->getDeliveryExecution($result['deliveryResultIdentifier']);
-                $row = array();
+                $row = [];
 
-                $startime = $this->cleanTimestamp($execution->getStartTime());
-                if (!$this->withinDateRange($startime)) {
+                $starTime = $this->cleanTimestamp($execution->getStartTime());
+                if (!$this->withinDateRange($starTime)) {
                     continue;
                 }
 
                 $user = $userService->getUserById($execution->getUserIdentifier());
 
                 $row['ID'] = $this->getUserId($user);
                 $row['IDFORM'] = $delivery->getLabel();
-                $row['STARTTIME'] = $startime;
+                $row['STARTTIME'] = $starTime;
                 $row['FINISHTIME'] = (($endTime = $execution->getFinishTime()) !== null) ? $this->cleanTimestamp($endTime) : '';
-    
-                $attemptScore = 0;
-
-                $callIds = $storage->getRelatedItemCallIds($execution->getIdentifier());
-                foreach ($callIds as $callId) {
 
-                    $itemResults = $storage->getVariables($callId);
-                    $splitCallId = explode('#', $callId);
+                $itemCallIds = $storage->getRelatedItemCallIds($execution->getIdentifier());
+                foreach ($itemCallIds as $itemCallId) {
+                    $itemResults = $storage->getVariables($itemCallId);
+                    $splitCallId = explode('#', $itemCallId);
                     $itemIdentifier = explode('.', $splitCallId[1]);
```

Comment:
We don't expect more than 2 parts here I assume.
```suggestion
                    $splitCallId = explode('#', $itemCallId, 2);
                    $itemIdentifier = explode('.', $splitCallId[1], 2);
```

Reply:
example of $itemCallId 
```
https://bosa.docker.localhost/ontologies/tao.rdf#i5f68bbb02e09e304856d61d15c4c60e18.item-9.0
```
$itemIdentifier should contain `item-9` but after adding `2` it becomes `item-9.0` what removes columns with scores from the report